openedx/features/edly/utils.py

```diff
@@ -147,17 +147,18 @@ def create_user_link_with_edly_sub_organization(request, user):
         user (object): User object.
 
     Returns:
-        object: EdlyUserProfile object.
+        object: EdlyUserProfile object or None.
 
     """
+    edly_user_profile = None
     try:
         edly_sub_org = request.site.edly_sub_org_for_lms
     except EdlySubOrganization.DoesNotExist:
         edly_sub_org = request.site.edly_sub_org_for_studio
-    edly_user_profile, __ = EdlyUserProfile.objects.get_or_create(user=user)
-    edly_user_profile.edly_sub_organizations.add(edly_sub_org)
-    edly_user_profile.save()
-
+    if edly_sub_org:
```

Comment:
Similarly, I think these changes are about something else and should be done in a separate PR.